deny.toml

```diff
@@ -0,0 +1,239 @@
+# This template contains all of the possible sections and their default values
```

Comment:
This file is huge and mostly comments. 

1. it shouldn't be in this PR
2. it shouldn't be so verbose
3. it permits licenses which are incompatible with the project

See comments below; let's remove this from the current PR and address it in another.